modules/api-support-tiers.adoc

```diff
@@ -0,0 +1,130 @@
+
+[discrete]
+[id="api-support-tier-1_{context}"]
+== Support tier 1
+APIs and application operating environments (AOEs) are stable within a major release for a minimum of 12 months or 3 minor releases from the announcement of deprecation, whichever is longer. The release that introduces a new or revised API or AOE, and the two following minor releases (n, n+1, n+2).
+
+[discrete]
+[id="api-support-tier-2_{context}"]
+== Support tier 2
+APIs and AOEs are stable within a major release for a minimum of 9 months or 3 minor releases from the announcement of deprecation, whichever is longer. Compatibility level 2 is the default for components in {product-title} where not identified.
+
+[discrete]
+[id="api-support-tier-3_{context}"]
+== Support tier 3
+This level applies to languages, tools, applications, and optional Operators included with {product-title} through Operator Hub.  Each component will specify a lifetime during which the API and AOE will be supported. Newer versions of language runtime specific components will attempt to be as API and AOE compatible from version to version as possible. Version to version compatibility is not guaranteed, however.
+
+Components and developer tools that receive continuous updates through the Operator Hub, referred to as Operators and operands, should be considered compatibility level 3.  Developers should use caution and understand how these components may change with each release.  Users are encouraged to consult the compatibility guidelines documented by the component.
+
+[discrete]
+[id="api-support-tier-4_{context}"]
+== Support tier 4
+No compatibility is provided. API and AOE can change at any point. These capabilities should not be used by applications needing long-term support.
+
+It is common practice for Operators to use Custom Resource Definitions (CRDs)  internally to accomplish a task.  These objects are not meant for use by actors external to the Operator and are intended to be hidden.  If any CRD is not meant for use by actors external to the Operator, the `operators.operatorframework.io/internal-objects` annotation in the Operators `ClusterServiceVersion` (CSV) must be specified, and that signals that the corresponding resource is internal use only.
+
+[id="mapping-support-tiers-to-api-groups_{context}"]
+= Mapping support tiers to API groups
+
+For each support tier defined by Red Hat, we provide a mapping table for specific API groups where the upstream communities are committed to maintain forward compatibility. Any API group that does not specify an explicit compatibility level is assigned level 3 by default.
+
+[id="mapping-support-tiers-to-kubernetes-api-groups_{context}"]
+== Support for Kubernetes API groups
+
+API groups that end with the suffix `*.k8s.io `or have the form `version.<name>` with no suffix are governed by the Kubernetes deprecation policy and follow a general mapping between API version exposed and corresponding support tier unless otherwise specified.
```

Comment:
I wonder about `version.<name>`, because I haven't seen this before. I have seen for example `apps/v1`.